src/routes/data-decode/entities/get-data-decoded.dto.entity.ts

```diff
@@ -1,7 +1,6 @@
 import { ApiProperty } from '@nestjs/swagger';
-import { isHex } from '../../common/utils/utils';
 
-export class CreateDataDecodedDto {
+export class GetDataDecodedDto {
```

Comment:
Moving `isCreateDataDecodeDto` guard/validation function from this entity class to the proper ValidationPipe.